### PR TITLE
Feature/gem upgrades

### PR DESCRIPTION
**Pull request by sapieneptus**
Created `calabash-sandbox update` command for updating gems using bundler. 

Works by `cd`ing into the `~/.calabash/sandbox` dir, fetching `Gemfile.OSX` from github, and running `bundle update` from within the sandbox.

CC: @topgenorth @krukow @jmoody

### CODE REVIEW

File: calabash-sandbox [hunk 1]
```diff
@@ -1,15 +1,42 @@
...
+SANDBOX="${HOME}/.calabash/sandbox"
+THIS_SCRIPT=$(basename $0)
 
 if [ "$(uname -s)" != "Darwin" ]; then
   echo "Calabash-sandbox only runs on Mac OSX"
   exit 1
 fi
 
-if [ $# == 1 ]; then
+if [ "${1}" == "version" ]; then
```
Comment (jmoody) [inline calabash-sandbox]: `==` works, but the convention is to use `=` to compare strings.  There are some platforms, none of which we work on, where `==` will not work.

**Comment by jmoody** [comment]
Wow. This is hard to test.

I found this:

```
$ ./calabash-sandbox update
Updating gems...
bash: /Users/moody/.calabash/sandbox/Gems/bin/bundle: /Users/chrisf/.rvm/rubies/ruby-2.0.0-p643/bin/ruby: bad interpreter: No such file or directory
```

**Comment by sapieneptus** [comment]
Yeah, I saw that as well - I'm not sure what that's about, it seems something may be wrong with the `bundle` I am distributing...

**Comment by sapieneptus** [comment]
@jmoody 
Try this:
```
DEV=1 DEBUG=1 SANDBOX_URL=https://raw.githubusercontent.com/calabash/install/feature/gem-upgrades/calabash-sandbox ./install-osx.sh 
```

Post merge, the sandbox_url will point to the one in Github Master branch

**Comment by sapieneptus** [comment]
Also, @jmoody I have updated `bundle` so it uses `#!/usr/bin/env ruby`. Bloody rvm

**Comment by jmoody** [comment]
```shell
$ calabash-sandbox update
...
Using calabash-cucumber 0.17.0 (was 0.16.4)
Using rest-client 1.8.0
Using xamarin-test-cloud 1.1.2
Bundle updated!

Done! Now the sandbox contains:
calabash-ios:       0.17.0
calabash-android:   0.5.15
xamarin-test-cloud: 1.1.2
```

**Comment by jmoody** [comment]
@sapieneptus Merge at will.

**Pull request merged**